agent/handlers/taskmetadata/taskinfo.go

```diff
@@ -46,6 +49,11 @@ func metadataV2Handler(state dockerstate.TaskEngineState, cluster string) func(h
 			return
 		}
 
+		if containerID := getContainerID(r.URL); containerID != "" {
+			writeContainerResponse(w, containerID, state)
+			return
+		}
+		seelog.Infof("V2 metadata: handling request for task '%s'", taskARN)
```

Comment:
Refactoring suggestion: 
```
if containerID := getContainerID(r.URL); containerID != "" {
	writeContainerResponse(w, containerID, state)
 	return
      }
writeTaskResponse(w, taskARN, state, cluster)
```

Reply:
I don't get it. What is it refactoring?

Reply:
Oh, the suggestion was to move the code block that writes the task response, to a method of its own (similar to `writeContainerResponse`).

Reply:
will have it in the next pr!